azurerm/resource_arm_cosmos_db_account.go

```diff
@@ -178,6 +178,7 @@ func resourceArmCosmosDBAccount() *schema.Resource {
 							Required:         true,
 							StateFunc:        azureRMNormalizeLocation,
 							DiffSuppressFunc: azureRMSuppressLocationDiff,
+							ValidateFunc:     validation.NoZeroValues,
```

Comment:
I think we should be able to use `locationSchema()` here?

Reply:
Would it make sense to add `NoZeroValues` validation to `locationSchema()` or can leaving a location property empty be valid?